pkgs/os-specific/linux/intel-sgx/sgx1/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, kernel, fetchurl, fetchFromGitHub }:
+assert lib.versionAtLeast kernel.version "5.10";
+
+stdenv.mkDerivation rec {
+  pname = "intel-sgx-sgx1";
+  version = "master";
+
+  src = fetchFromGitHub {
+    owner = "intel";
+    repo = "linux-sgx-driver";
+    rev = "0373e2e8b96d9a261657b7657cb514f003f67094";
```

Comment:
What is the difference to the sgx driver already packaged? This version will be not compatible with our kernel.

Reply:
I wasn't aware that there was already a driver, good catch! I'll see if the existing one works with my system.

Reply:
Yep, it works fine. Will just use that.